libsolidity/interface/OptimiserSettings.h

```diff
@@ -59,6 +59,8 @@ struct OptimiserSettings
 		"]"
 		"jmuljuljul VcTOcul jmul";     // Make source short and pretty
 
+	static size_t constexpr DefaultExecutions = 200;
```

Comment:
Why not use ``OptimiserSettings{}.expectedExecutionsPerDeployment` directly? Or ``OptimiserSettings::standard().expected...`?

Reply:
I could if you prefer that version. I thought that having a constant would be "cleaner" but I guess either version would ultimately just be optimized out into a constant so the question is just which version you prefer in terms of style.

Reply:
Constant is fine, but I'm mostly a bit worried about the name.

Reply:
In that case I think prefer dropping the constant because the name I'd use (`OptimiserSettings::DefaultExpectedExecutionsPerDeployment`) is even longer than just `OptimiserSettings{}.expectedExecutionsPerDeployment`.

Reply:
Done. Replaced with `OptimiserSettings{}.expectedExecutionsPerDeployment`.